### PR TITLE
Fixed issue #7189

### PR DESCRIPTION
**Pull request by akathuria289**
## Issue Reference
#7189

Screenshot from 2023-02-01 11-44-32

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
**Not working**, please find the screenshot [Here](https://prnt.sc/43fmHRaE8v_A)

**Comment by akathuria289** [comment]
@Sanjaybhattwebkul Please check now

**Comment by akathuria289** [comment]
@Sanjaybhattwebkul Now Check

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
**Not working**. please find the screenshot [Here](https://webkul.chatwhizz.com/share/screenshot/63eb70f593ea2803a9444ef1)

**Comment by akathuria289** [comment]
@Sanjaybhattwebkul This is already done in the PR - https://github.com/bagisto/bagisto/pull/6811

**Comment by devansh-webkul** [comment]
Closing this as per this comment,
https://github.com/bagisto/bagisto/pull/7251#issuecomment-1429629587

**Comment by Sanjaybhattwebkul** [comment]
@abdulhamid-alattar 
This is working fine.

**Review by Sanjaybhattwebkul — APPROVED** [review]

File: packages/Webkul/Velocity/src/Resources/assets/js/UI/components/sidebar.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
         :class="`sidebar ${addClass ? addClass : ''}`"
         v-if="slicedCategories && slicedCategories.length > 0"
     >
-        <ul type="none" style="margin-bottom: 0">
+        <ul type="none" class="main-cat">
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/js/UI/components/sidebar.vue:8]: Give some meaningful class names. Don't use the short version.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Review by devansh-webkul — DISMISSED** [review]

**Comment by devansh-webkul** [comment]
Conflicts.

File: packages/Webkul/Velocity/src/Resources/assets/js/UI/components/sidebar.vue (hunk 1 above)
Comment (akathuria289) [reply packages/Webkul/Velocity/src/Resources/assets/js/UI/components/sidebar.vue]: Done

**Comment by akathuria289** [comment]
@devansh-webkul Sir Resolved

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**